src/components/ContextualMenu/contextualmenu.js

```diff
@@ -0,0 +1,38 @@
+import React, {useState} from 'react';
+import './contextualmenu.css';
+import EditMovieModal from '../Modals/EditMovieModal/editmoviemodal';
+import DeleteMovieModal from '../Modals/DeleteMovieModal/deletemoviemodal';
+
+const ContextualMenu = ({ showMenu, xPos, yPos }) => {
+  
+  const [editShowModal, setEditShowModal] = useState(false);
+  const [deleteShowModal, setDeleteShowModal] = useState(false);
+  
+  const handleEditClose = () => setEditShowModal(false);
+  const handleEditShow = () => setEditShowModal(true);
+  
+  const handleDeleteClose = () => setDeleteShowModal(false);
+  const handleDeleteShow = () => setDeleteShowModal(true);
+  
+  if (deleteShowModal) {
+    return <DeleteMovieModal show={deleteShowModal} handleClose={handleDeleteClose} />;
+  }
+  if (editShowModal) {
+    return <EditMovieModal show={editShowModal} handleClose={handleEditClose} />;
+  }
+  if (showMenu) {
```

Comment:
I think that `showMenu` logic should be handled in the parent component and if the `showMenu` is false the component shouldn't be rendered at all

Reply:
fixed